### PR TITLE
Fix alignment of tip button on Twitter.

### PR DESCRIPTION
**Pull request by mkarolin**
Fixes https://github.com/brave/brave-browser/issues/26880
Fixes https://github.com/brave/brave-browser/issues/27256

### CODE REVIEW

File: scripts/brave_rewards/publisher/twitter/tipping.ts [hunk 1]
```diff
@@ -279,14 +266,17 @@ export const configure = () => {
     let actions
 
     if (newTwitter) {
-      if (isThreadParent(tweets[i])) {
-        const replyButton = tweets[i].querySelector('[aria-label="Reply"]')
-        if (replyButton) {
-          actions = replyButton.parentElement?.parentElement
-        }
+      const groups = tweets[i].querySelectorAll('[role="group"]')
+      for (let j = 0; j < groups.length; ++j) {
```
Comment (zenparsing) [inline scripts/brave_rewards/publisher/twitter/tipping.ts:270]: Looks like this could use for-of:

```js
for (let group of groups) {
  ...
}
```

**Review by zenparsing — APPROVED** [review]

File: scripts/brave_rewards/publisher/twitter/tipping.ts (hunk 1 above)
Comment (mkarolin) [reply scripts/brave_rewards/publisher/twitter/tipping.ts]: Thanks! Updated.

**Review by zenparsing — APPROVED** [review]

**Review by emerick — APPROVED** [review]
LGTM

**Comment by mkarolin** [comment]
Updated one more time. As it turns out `aria-label`'s value is localized so the previous solution would only work in English and then would just fallback on the last `group`. Changed to look for a `group` of more than 3 `button`s.

**Review by emerick — APPROVED** [review]

**Review by szilardszaloki — APPROVED** [review]

**Pull request merged**